relay/paseo/src/weights/frame_system.rs

```diff
@@ -47,6 +47,8 @@ use core::marker::PhantomData;
 /// Weight functions for `frame_system`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
+	fn authorize_upgrade() -> frame_election_provider_support::Weight { todo!() }
+fn apply_authorized_upgrade() -> frame_election_provider_support::Weight { todo!() }
```

Comment:
TODO

Reply:
thanks for letting us know. Indeed, today i will run the benchmark for this the frame system pallet :)